modules/repartitioner.py

```diff
@@ -64,6 +90,24 @@ def editPartition(self, targetPartition, deltaStart, deltaEnd, fstype, fslabel,
                 log.error("movePartition: Format of " + fstype + " is not implemented.")
                 return False
 
+        # Restore data in targetPartition
+        if safeDataThroughTmp:
+            # Make sure targetPartition is mounted in mountpoint
+            if isMounted(targetPartition.path):
+                log.error("editPartition: Something is wrong. %s should not be mounted." % targetPartition.path)
+                return False
+            else:
+                if not mount(targetPartition.path, mountpoint):
```

Comment:
I got it. I just let the code flow if not. Cause I have another block before which does the backup.